dcos_install_centos7.sh

```diff
@@ -13,7 +13,8 @@
 #CUSTOMER_KEY="customer_key: <insert customer key here and uncomment this line>"
 USERNAME=bootstrapuser
 PASSWORD=deleteme
-DOWNLOAD_URL="https://downloads.dcos.io/dcos/EarlyAccess/dcos_generate_config.sh"
+DCOS_VERSION="EarlyAccess"
```

Comment:
Might want to call this something else than `version` since technically the input here would be `testing/pull/<pr-number>` as we do in CCM to get different "version"s.